src/ApiService/ApiService/QueueProxyUpdate.cs

```diff
@@ -0,0 +1,25 @@
+using System;
+using Microsoft.Azure.Functions.Worker;
+using Microsoft.Extensions.Logging;
+using System.Text.Json;
+using Microsoft.OneFuzz.Service.OneFuzzLib.Orm;
+
+namespace Microsoft.OneFuzz.Service;
+
+public class QueueProxyUpdate 
+{
+    private readonly ILogger _logger;
+
+    public QueueProxyUpdate(ILoggerFactory loggerFactory)
+    {
+        _logger = loggerFactory.CreateLogger<QueueProxyUpdate>();
+    }
+
+    [Function("QueueProxyUpdate")]
+    public void Run([QueueTrigger("myqueue-items", Connection = "AzureWebJobsStorage")] string msg)

```

Comment:
this needs to be update to match the config in host.json

Reply:
Yes was just doing this for testing!